index.html

```diff
@@ -8,343 +8,328 @@
 	  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     <title>Supreeti Portfolio</title>
   </head>
-  <body>
-    <header class="header1">
+  <body class="bg1">
+    
+    <header class="bg2">
+    <div class="header1">
         <h3>My Logo</h3>
-          <nav>
+          <nav class="nav1">
               <img src="images/Union.png" alt=""/>
           </nav>
-    </header>
+          <nav class="nav2">
+            <ul class="nav2 liststyle">
+                <li class="list_style"> <a class="textdecoration" href="#">Portfolio</a></li>
+                <li class="list_style"> <a class="textdecoration" href="#">About</a></li>
+                <li class="list_style"><a class="textdecoration" href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </div>
+     </header>
```

Comment:
[OPTIONAL] 

- KIndly fix the content of your header section to match the figma design. 

YOur design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213882845-10289193-74dc-4c6c-b0fe-2aab7e5d6b7f.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/213883018-f88cd40b-4256-437d-a403-6248fb918510.png)